tendrl/node_agent/node_sync/disk_sync.py

```diff
@@ -79,9 +78,21 @@ def get_node_disks():
                         else:
                             rv['used_disks_id'].append(disk['disk_id'])
         else:
-            LOG.error(err)
+            Event(
+                Message(
+                    priority="error",
+                    publisher=tendrl_ns.publisher_id,
+                    payload={"message": err}
+                )
+            )
```

Comment:
Make sure this is 'err' is string, other wise it gives  serialization problem, If Exception message class came then you can use exception as it is. But now you have to use string only.